### DISCUSSION TITLE
Rights permissions on Windows filesystem

### DISCUSSION THREAD

**Discussion by LaetitiaRedFroggy** [dadoonet/fscrawler#1313] [Ideas]
Hello,

Is there a way to extract rights informations on Windows filesystem ? I'm going to start a project where I will manage the rights, I wanted to know beforehand if something had already been done

Thanks in advance

**Comment by dadoonet** [comment]
Are you looking for https://github.com/dadoonet/fscrawler/issues/464?

**Reply by LaetitiaRedFroggy** [reply]
Hello David, thank you, I saw that ACL features are not supported yet. When we will start our new project, we will have to work on this feature, I'll let you know if we can contribute on the project.

**Comment by helsonxiao** [comment]
Looking for this feature too. We've almost finished the crawler for Windows SMB except the rights/permission part.
